src/locales/en.json

```diff
@@ -31,13 +31,12 @@
     "round": "Round",
     "arrowheads": "Arrowheads",
     "arrowhead_none": "None",
-    "arrowhead_arrow": "Arrow",
```

Comment:
Same meaning with `toolBar.arrow`.. no need to duplicate

Reply:
It may work now, but I'd generally not reuse unrelated labels. Moreoever, we can't guarantee that in some languages they have different terms for "arrow" as a whole, and "arrow" as an arrow cap. As such, I'd leave it as is.